### PR TITLE
Revert liquid removable updates to fix water issues

### PR DESCRIPTION
**Pull request by JustTalDevelops**

### CODE REVIEW

**Review by provsalt — APPROVED** [review]

**Comment by JustTalDevelops** [comment]
Forgot to mention that I switched it back to an array & fixed the actual issue which was causing the tsunami-lime issues. Ready for a review, let me know if there is anything I should change.

File: dragonfly/world/block.go [hunk 1]
```diff
@@ -189,6 +189,7 @@ func RegisterBlock(b Block, s BlockState) error {
 	if !ok {
 		return fmt.Errorf("block state returned is not currently registered (%+v)", s)
 	}
+	fmt.Println(s.Name, rid)
```
Comment (Sandertv) [inline dragonfly/world/block.go:192]: Make sure to remove this.

Comment (JustTalDevelops) [reply dragonfly/world/block.go]: Removed, completely forgot about this lmao

**Comment by JustTalDevelops** [comment]
Removed everything asked, let me know if I should change anything else

**Review by Sandertv — APPROVED** [review]

**Pull request merged**